spec/lib/administrate/resource_resolver_spec.rb

```diff
@@ -7,23 +7,23 @@
   describe "#dashboard_class" do
     it "handles global-namepsace models" do
       begin
-        class UserDashboard; end
-        resolver = Administrate::ResourceResolver.new("admin/users")
+        class AccountDashboard; end
+        resolver = Administrate::ResourceResolver.new("admin/accounts")
 
-        expect(resolver.dashboard_class).to eq(UserDashboard)
+        expect(resolver.dashboard_class).to eq(AccountDashboard)
       ensure
-        remove_constants :UserDashboard
+        remove_constants :AccountDashboard
```

Comment:
It's been a while, but it seems like a premature improvement.

I will do this to the way as it was.